controls/radgridview/styles-and-templates/styling-a-row.md

```diff
@@ -10,7 +10,9 @@ position: 14
 
 # Styling Rows
 
->tipBefore reading this topic, you might find it useful to get familiar with the [Template Structure of the GridViewRow]({%slug radgridview-styles-and-templates-templates-structure%}#gridviewrow).
+>tipBefore reading this topic, you might find it useful to get familiar with the [Template Structure of the GridViewRow]({%slug radgridview-styles-and-templates-templates-structure%}#gridviewrow).
+
+> As of __R3 2018 RadGridView__ supports setting the __Background__ of the selected row. More information can be found [here]({%slug radgridview-rows-selected-background%}).
```

Comment:
"background"

Also, you can change the link word "here" to something meaningful like "Selected Row Background" - More information can be found in the Selected Row Background article.